src/js/router.coffee

```diff
@@ -406,17 +406,12 @@ class Router extends Backbone.Router
         assessment = new Assessment "_id" : id
         assessment.deepFetch
           success : ->
-            Tangerine.assessment = assessment
-            viewOptions =
-              assessment: Tangerine.assessment
             dashboardLayout = new DashboardLayout();
```

Comment:
@chrisekelley It looks like we're not taking full advantage of DashboardLayout here. Shall we remove it from this route?